README.md

```diff
@@ -16,6 +16,7 @@ See also [free database services](https://github.com/DmitryScaletta/free-databas
 | Adaptable.io | Yes* | No  | ~25,000 req/mo | 256MB | 1GB    | Yes* | 5GB           | No* | Yes |
 | Zeabur.com   | No   | No  | $5             | 512MB | 1GB    | Yes  |               | Yes | Yes |
 | Koyeb.com    | Yes* | No  |                | 512MB | 2GB    | Yes  | 100GB         | Yes | Yes |
+| Lade.io      | No   | No  |                | 128MB | 1GB    | Yes  | 100GB         | Yes | No  |
```

Comment:
Thanks for the PR.
Can you please provide the source where you got the information about Network Bandwidth 100GB?

Reply:
If you visit https://www.lade.io/pricing the last FAQ is below:

### How much does bandwidth cost?

Each service comes with 100 GB of free outbound bandwidth per month. Additional outbound bandwidth above the free quota is charged at $0.05 per GB.